src/bindings/python/src/pyopenvino/core/properties/properties.cpp

```diff
@@ -65,6 +64,11 @@ void regmodule_properties(py::module m) {
     wrap_property_RW(m_hint, ov::hint::model, "model");
     wrap_property_RW(m_hint, ov::hint::allow_auto_batching, "allow_auto_batching");
 
+    // Submodule intel_cpu
+    py::module intel_cpu = m_properties.def_submodule("intel_cpu",
+        "openvino.runtime.properties.intel_cpu submodule that simulates ov::intel_cpu");
+    wrap_property_RW(intel_cpu, ov::intel_cpu::denormals_optimization, "denormals_optimization");
```

Comment:
```suggestion
    py::module m_intel_cpu = m_properties.def_submodule("intel_cpu",
        "openvino.runtime.properties.intel_cpu submodule that simulates ov::intel_cpu");
    wrap_property_RW(m_intel_cpu, ov::intel_cpu::denormals_optimization, "denormals_optimization");
```
Just to suggest it is a module.

Reply:
done, thanks.